andela/mmdp_site
# MMDP-529 - View Stakeholders by LGA

**What does this PR do?**
Allow an administrator to view Stakeholders as per the specific LGA

**description of Task to be completed?**
- [x] View Number of Stakeholders
- [x] View Number of Focus Area
- [x] List Stakeholder's name and email address in a table
- [x] Paginate Stakeholder details in an LGA

**How should this be manually tested?**
- check out to this branch, `ft-view-stakeholders-in-lga-529`.
- start development server, `yarn dev`.
- Using the sidebar menu in your browser, click on the state level and click View a lga.Make sure you have uploaded SVG Maps. 

**Any background context you want to provide?**

- None

**What are the relevant Jira issues?**

[MMDP-529](https://viisaus.atlassian.net/browse/MMDP-529)

**Screenshots**
<img width="1494" alt="Screenshot 2019-05-23 at 11 20 55" src="https://user-images.githubusercontent.com/25298492/58237025-df5e5a80-7d4c-11e9-9571-952d059d3eb9.png">



**Questions:**

- None
